src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -115,6 +115,7 @@ private readonly Dictionary<RestoreTargetGraph, Dictionary<string, LibraryInclud
         private const string AuditTransitiveVulnerabilitySev2 = "Audit.Vulnerability.Transitive.Severity2";
         private const string AuditTransitiveVulnerabilitySev3 = "Audit.Vulnerability.Transitive.Severity3";
         private const string AuditTransitiveVulnerabilitySevInvalid = "Audit.Vulnerability.Transitive.SeverityInvalid";
+        private const string AuditPackageDownloadVulnerabilitiesPackages = "Audit.Vulnerability.PackageDownload.Packages";
```

Comment:
Did you and @zivkan have an extra conversation that wasn't included here?

Curious why we're not adding the counts for package downloads with vulnerabilities?

Reply:
Not that I can remember 🤣 

I think `Audit.Vulnerability.PackageDownload.*` properties make sense and will make it easier for us to tell the difference between older versions of NuGet/VS that don't support this telemetry, and restores that have zero PackageDownloads with known vulnerabilities.  We could do it by devenv ExeVersion, but I prefer to filter out null values.

Reply:
I was thinking we could do the counting from the `AuditPackageDownloadVulnerabilitiesPackages` list.